test/general-spec.js

```diff
@@ -129,6 +129,68 @@ describe('Functionality that is common to all documents:', function() {
     });
   });
 
+  describe('validator of supported properties', function() {
```

Comment:
The specs for the sample document definitions indirectly test these behaviours as well, but this provides a more systematic means of verifying.